src/test/java/com/epam/tc/hw2/ex1/GithubPageSeleniumTestsEx1.java

```diff
@@ -19,55 +26,46 @@ public void homePageTest() {
 
         // 3. Perform login
         webDriverWait.until(ExpectedConditions.elementToBeClickable(
-            webDriver.findElement(By.xpath("//*[@class='profile-photo'][1]")))).click();
+            webDriver.findElement(By.id("user-icon")))).click();
         webDriver.switchTo().activeElement();
         webDriverWait.until(ExpectedConditions.visibilityOf(
-            webDriver.findElement(By.xpath("//input[@id='name'][1]")))).sendKeys("Roman");
+            webDriver.findElement(By.id("name")))).sendKeys("Roman");
         webDriverWait.until(ExpectedConditions.visibilityOf(
-            webDriver.findElement(By.xpath("//input[@id='password'][1]")))).sendKeys("Jdi1234");
+            webDriver.findElement(By.id("password")))).sendKeys("Jdi1234");
         webDriverWait.until(ExpectedConditions.elementToBeClickable(
-            webDriver.findElement(By.xpath("//button[@id='login-button'][1]")))).click();
+            webDriver.findElement(By.id("login-button")))).click();
 
         // 4. Assert Username is loggined
         String userName = webDriverWait.until(ExpectedConditions.visibilityOf(
-            webDriver.findElement(By.xpath("//span[@id='user-name']")))).getText();
+            webDriver.findElement(By.id("user-name")))).getText();
         soft.assertThat(userName).isEqualTo("ROMAN IOVLEV");
 
         // 5. Assert that there are 4 items on the header section are displayed and they have proper texts
-        webDriverWait.until(ExpectedConditions.visibilityOfAllElements(
-                webDriver.findElements(By.xpath("//ul[@class='uui-navigation nav navbar-nav m-l8'][1]/li"))));
-        List<WebElement> menuTabs = webDriver.findElements(
-            By.xpath("//ul[@class='uui-navigation nav navbar-nav m-l8'][1]/li"));
-        soft.assertThat(menuTabs.size()).isEqualTo(4);
-        soft.assertThat(menuTabs.get(0).getText()).isEqualTo("HOME");
-        soft.assertThat(menuTabs.get(1).getText()).isEqualTo("CONTACT FORM");
-        soft.assertThat(menuTabs.get(2).getText()).isEqualTo("SERVICE");
-        soft.assertThat(menuTabs.get(3).getText()).isEqualTo("METALS & COLORS");
+        List<WebElement> menuTabs = webDriverWait.until(ExpectedConditions.visibilityOfAllElements(
+            webDriver.findElements(By.xpath("//ul[@class='uui-navigation nav navbar-nav m-l8'][1]/li"))));
```

Comment:
css is preferable over xpath
```suggestion
            webDriver.findElements(By.css("header ul.navbar-nav.m-l8 > li"))));
```